app/options.go

```diff
@@ -38,8 +39,19 @@ func StoreOverride(override interface{}) Option {
 
 func ConfigFile(file string, watch bool) Option {
```

Comment:
The various `Option` returning functions in this file are used both at startup and during tests for configuring behaviour. `ConfigFile` preserves the current interface, while `ConfigStore` is more flexible, allowing for a `MemoryStore` or possibly the `DatabaseStore` as required in the future.